gstreamer-pbutils/src/auto/install_plugins_context.rs

```diff
@@ -0,0 +1,61 @@
+// This file was generated by gir (d933f9a+) from gir-files (???)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct InstallPluginsContext(Boxed<ffi::GstInstallPluginsContext>);
```

Comment:
This alone is not very useful without all the other missing plugin API. I would skip that for now unless you need it.